### DISCUSSION TITLE
window.addeventlistener not working

### DISCUSSION THREAD

**Discussion by giglr** [nuxt/nuxt#24264] [Questions]
hello i want to make a sticky header using vue, nuxt and typescript. i want the header to disappear when scrolling up and appear when scrolling down. i coded it but window.addeventlistener does not work. it dynamically disappears and does not appear. it does not work even if i do not get an error

**Comment by TheAlexLichter** [comment]
You might want to use it in `onMounted` instead (because [`window` is not a thing on SSR](https://www.lichter.io/articles/universal-ssr-vue-component-guide/)).

To get better help, kindly provide a [reproduction](https://nuxt.com/docs/community/reporting-bugs/#create-a-minimal-reproduction)
